meteo-notebook.ipynb

```diff
@@ -110,7 +110,7 @@
   },
```

Comment:
<p>Using a line/bar chart instead of scatter, would be better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>made it a line chart</p>